InnerEye/ML/Histopathology/preprocessing/loading.py

```diff
@@ -88,6 +94,7 @@ def _get_bounding_box(self, slide_obj: CuImage) -> Tuple[box_utils.Box, float]:
         return bbox, threshold
 
     def __call__(self, data: Dict) -> Dict:
+        from cucim import CuImage
```

Comment:
This has already been imported at the top of the file - does it need to be imported again here?

Reply:
you're right, it does not need to be imported again.